src/AppInstallerRepositoryCore/Microsoft/ARPHelper.cpp

```diff
@@ -288,6 +302,14 @@ namespace AppInstaller::Repository::Microsoft
             //       is from it's ARP location, despite it very clearly being a specific architecture. And note that user
             //       scope does not have separate ARP locations, so every architecture would appear as native.
 
+            // Publisher is needed for certain scenarios but we don't store it from the manifest

```

Comment:
>don't store it [](start = 64, length = 14)

do need to store?

Reply:
We only keep name in the index for searching.  Publisher isn't really needed there, but it needs to flow from ARP, so it goes in the metadata.